src/EPR.Calculator.Frontend/Controllers/LocalAuthorityUploadFileProcessingController.cs

```diff
@@ -29,6 +31,8 @@ public IActionResult Index([FromBody] List<LapcapDataTemplateValueDto> lapcapDat
                     throw new ArgumentNullException(lapcapSettingsApi, "LapcapSettingsApi is null. Check the configuration settings for local authority");
                 }
 
+                this.FileName = this.TempData.Peek("FileName").ToString();
```

Comment:
The same fileUpload for Lapcap is done in a different way to this. 
We should follow one standard way to read the File please. Can you do your changes so that this code aligns to Lapcap? Thanks